### ISSUE TITLE
[Bug]: Low Gtk texte entry but only with flatpak apps

### ISSUE THREAD

**Issue by LorierMatthieu** [flatpak/flatpak#6029]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a bug that matches the one I want to file, without success.
- [X] If this is an issue with a particular app, I have tried filing it in the appropriate issue tracker for the app (e.g. under https://github.com/flathub/) and determined that it is an issue with Flatpak itself.
- [X] This issue is not a report of a security vulnerability (see [here](https://github.com/flatpak/flatpak/blob/main/SECURITY.md) if you need to report a security issue).

### Flatpak version

1.14.6

### What Linux distribution are you using?

Ubuntu

### Linux distribution version

6.8.0-49-generic #49-Ubuntu SMP PREEMPT_DYNAMIC Mon Nov  4 02:06:24 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

### What architecture are you using?

x86_64

### How to reproduce

1 - Launch a apps installed with Flatpak (Gnome Builder for example)
2 - Try to enter some texte quickly with the keyboard

### Expected Behavior

The text just write with the letters in the right order.

### Actual Behavior

 All letters blend together.

https://github.com/user-attachments/assets/f602dc06-0445-4f9e-a9fd-2a8040ef2425

**Comment by chrisawi** [comment]
This is unlikely to have anything to do with Flatpak itself.  It may be https://github.com/ibus/ibus/issues/2486 which should be fixed in ibus 1.5.29, but it appears you may already have that. Is this Ubuntu 24.04?

**Comment by LorierMatthieu** [comment]
Yes, I'm using Ubuntu 24.04 LTS. My ibus version is IBus 1.5.29-rc2.
